oauth/v1/types.go

```diff
@@ -37,6 +37,9 @@ type OAuthAccessToken struct {
 
 	// RefreshToken is the value by which this token can be renewed. Can be blank.
 	RefreshToken string `json:"refreshToken,omitempty" protobuf:"bytes,9,opt,name=refreshToken"`
+
+	// TimeoutsIn is the seconds from CreationTime before this token timeouts.
+	TimeoutsIn int32 `json:"timeoutsIn,omitempty" protobuf:"varint,10,opt,name=timeoutsIn"`
```

Comment:
Should be TokenExpirationSeconds

Reply:
`InactivityTimeoutSeconds`

Reply:
Better godoc here.  Explain what it actually does, and what happens when this value is reached.

Reply:
I am being consistent with ExpiresIn here, any good reason to diverge ?

Reply:
ExpiresIn at least is correct verb tense and grammar.  Also, ExpiresIn is consistent with oauth spec but predates our API conventions, which at this point are fairly strongly enforced.  Given TimeoutsIn isn't part of the oauth spec, making it consistent with our API conventions trumps existing fields.

Generally we

* always denote the units (`Seconds`)
* use full words to describe the thing this is `InactivityTimeout`
* combine them :)

Reply:
oook ... InactivityTimeoutSeconds it is ...